tests/conftest.py

```diff
@@ -49,6 +50,19 @@ def fx_tmpdir(tmpdir):
 
 
 @yield_fixture
+def fx_change_cwd_to_python_dir():
```

Comment:
`change_cwd` can be shortend to `cd`.  What do you guys think about this? @clicheio/owners